classes/04class/exercises/c04-iac03/christina-roperto/README.md

```diff
@@ -0,0 +1,12 @@
+# C04-IAC03
+
+## Command Execution Output
+- [output.txt](output.txt)
+
+## Questions:
+- Describe below why some changes just updated the resource being changed and the other required a full replacement of the resource.
+    - changed and the other required a full replacement of the resource.
+Because some resources in AWS can't be changed once they are created (eg: S3 bucket name, vpc and subnet cidr, nat gateway) and the only way to change them is to replace them (destroying the existing resources and creating new ones). When we replace those resources, the other resources that associated with them will subsequently need to be replaced as well.
```

Comment:
just out of curiosity.. you  can't change the vpc CIDR but you can add secondary ones, although you should always try to avoid doing that

https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html#vpc-resize